File: @LV1/1.4/babybites/babybites.rb
_ = gets
l = gets.split
cnt = 1
is_valid = true

l.each do |i|
  if i != 'mumble' && i != cnt.to_s
    is_valid = false
    break
  end
  cnt += 1
end

if is_valid
  puts 'makes sense'
else
  puts 'something is fishy'
end

File: @LV1/1.4/babybites/babybites.rs
use std::io::{self, BufRead};

fn main() {
    let stdin = io::stdin();
    let mut buff = stdin.lock().lines().map(|l| l.unwrap());
    buff.next();

    let mut cnt = 1;
    let mut is_valid = true;

    for l in buff.next().unwrap().split_whitespace() {
        if l != "mumble" {
            let n: i32 = l.parse().unwrap();
            if n != cnt {
                is_valid = false;
                break;
            }
        }
        cnt += 1;
    }

    if is_valid {
        println!("makes sense");
    } else {
        println!("something is fishy");
    }
}
